Apps/WebClient/src/ClientApp/public/index.html

```diff
@@ -21,6 +21,12 @@
 
         <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
         <script type="text/javascript">
+            var sBrowser, sUsrAg = window.navigator.userAgent;
+            if (sUsrAg.indexOf("Trident") > -1) {
+                console.info("Is Microsoft Internet Explorer - sUsrAg: " + sUsrAg);
+                alert("Health Gateway is not compatible with this browser. Please use another browser.");
```

Comment:
are you sure we want to display an alert? I thought it would be just a text in the page saying its not supported.

Reply:
yes, @ninosamson has confirmed on the https://qslvic.visualstudio.com/Health%20Gateway/_workitems/edit/9103
Also I think it is good and easy enough with an alert. But we can confirm with @ninosamson again on our call.

Reply:
What is harm of having alert vs page? 
@tiagograf-AOT is right though - I forgot the acceptance criteria was for page.

Reply:
I typically haven't seen an alert used in this situation as its quite intrusive.  Also, we shouldn't be using the UserAgent and we specifically mentioned making use of Modernizr what changed?

Reply:
having a message on the index page should be easy too.
Re. Modernizr integration, it is a blocker for me as I have tried many things without success to add it to our web-pack.

Reply:
@leobcgov Let's do the page with the message.